### PR TITLE
tasks: new task to clean up expired sessions

### PR DESCRIPTION
**Pull request by hachreak**
* Adds a new task to clean up expired sessions. (addresses #52)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: invenio_accounts/tasks.py [hunk 1]
```diff
@@ -40,3 +45,12 @@ def send_security_email(data):
     msg = Message()
     msg.__dict__.update(data)
     current_app.extensions['mail'].send(msg)
+
+
+@shared_task
+def clean_session_table():
+    """Clean session table."""
+    expired_moment = datetime.now() - current_app.permanent_session_lifetime
```
Comment (lnielsen) [inline invenio_accounts/tasks.py]: i think this should be ``datetime.utcnow()``

File: invenio_accounts/tasks.py [hunk 2]
```diff
@@ -40,3 +45,12 @@ def send_security_email(data):
...
+    SessionActivity.query.filter(
```
Comment (lnielsen) [inline invenio_accounts/tasks.py]: Perhaps we could add it as a method on SessionActivity...something like:

``SessionActivity.query_by_expired().delete()``

**Comment by hachreak** [comment]
Task documentation added!

**Comment by hachreak** [comment]
ping  @lnielsen

**Pull request merged**